frontend/app/src/components/history/transactions/events/TransactionEvents.vue

```diff
@@ -68,31 +67,60 @@ const headers: DataTableHeader[] = [
   }
 ];
 
-const events = computed<EthTransactionEventEntry[]>(() =>
-  get(transaction).decodedEvents!.map((event: EthTransactionEventWithMeta) => {
-    const { entry, ...entriesMeta } = event;
-    return {
-      ...entry,
-      ...entriesMeta
-    };
-  })
+const { fetchHistoryEvents } = useHistoryEvents();
+
+const evaluating: Ref<boolean> = ref(false);
+
+const num: ComputedRef<number> = computed(
+  () => get(eventGroupHeader).groupedEventsNum || 1
 );
 
-const editEvent = (item: EthTransactionEventEntry) => emit('edit:event', item);
-const deleteEvent = (item: EthTransactionEventEntry) =>
-  emit('delete:event', { txEvent: item, tx: get(transaction) });
+const events: Ref<HistoryEventEntry[]> = ref([get(eventGroupHeader)]);
+
+const fetchEvents = async () => {
+  set(evaluating, true);
+  const { data } = await fetchHistoryEvents({
+    limit: -1,
+    offset: 0,
+    eventIdentifiers: get(eventGroupHeader).eventIdentifier,
+    groupByEventIds: false
+  });
```

Comment:
What does this do? get all the events for the identifier?

Reply:
yes all events on each event group header